vm-setup/roles/libvirt/templates/ironic_nodes.json.j2

```diff
@@ -12,13 +12,18 @@
   {% for node in vm_nodes %}
     {
       "name": "{{ node.name|replace('_', '-') }}",
-      "driver": "ipmi",
+      "driver": "{{ vm_driver }}", 
       "resource_class": "baremetal",
       "driver_info": {
-        "ipmi_username": "admin",
-        "ipmi_password": "password",
-        "ipmi_address": "{{ lvars['host_ip'] }}",
-        "ipmi_port": "{{ node.virtualbmc_port }}",
+        "username": "admin",
+        "password": "password",
+        {% if vm_driver =='redfish' -%}
+        "port": "8000",
+        "address": "{{vm_driver}}+http://{{ lvars['host_ip'] }}:8000/redfish/v1/Systems/{{vm_id[node.name]}}",
+        {% else -%}
+        "port": "{{ node.virtualbmc_port }}",
+        "address": "{{vm_driver}}://{{lvars['host_ip']}}:{{node.virtualbmc_port}}",
```

Comment:
I wonder if we still need the "port" key, if it's going to be included in the uri value of "address" now?

Reply:
Agreed. Just kept it there for better visibility